Tools/ardupilotwaf/boards.py

```diff
@@ -1000,6 +1000,15 @@ def configure_env(self, cfg, env):
         if cfg.options.apstatedir:
             cfg.define('AP_STATEDIR', cfg.options.apstatedir)
 
+        defaults_file = f'libraries/AP_HAL_Linux/boards/{self.get_name()}/defaults.parm'
```

Comment:
unfortunately, I think that we still have the build server on python2 so no f-string

Reply:
it's been over a year, now... =[